tools/king/king.xml

```diff
@@ -1,33 +1,9 @@
-<tool id="king" name="KING" version="2.2.4" python_template_version="3.5">
+<tool id="king" name="KING" version="2.2.4">
     <requirements>
-	<requirement type="package">king</requirement>
+	<requirement type="package" version="2.2.4">king</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
-    king -b $genotype --fam $family --bim $map
-
-    #if $related
-    --related
-    #end if
-
-    #if $duplicate
-    --duplicate
-    #end if
-
-    #if $kinship
-    --kinship
-    #end if
-
-    #if $ibdseg
-    --ibdseg
-    #end if
-
-    #if $ibs
-    --ibs
-    #end if
-
-    #if $homog
-    --homog
-    #end if
+    king -b '$genotype' --fam '$family' --bim '$map' $related $duplicate $kinship $ibdseg $ibs $homog
```

Comment:
spreading out the optional arguments over newlines would look better.


Also it doesn't hurt to quote these arguments either